docs/user-guide/jobs.md

```diff
@@ -373,7 +373,3 @@ driver, and then cleans up.
 
 An advantage of this approach is that the overall process gets the completion guarantee of a Job
 object, but complete control over what pods are created and how work is assigned to them.
-
-## Future work
-
-Support for creating Jobs at specified times/dates (i.e. cron) is expected in [1.4](https://github.com/kubernetes/kubernetes/pull/11980).
```

Comment:
This conflicts with my PR #1466 but you made more useful changes in other places. Might be nice to still have a link at the bottom of the page to ScheduledJobs because they are still related to Jobs and no link exists on the page otherwise.